joomla/joomla-cms
# [4.0] Fix frontend module editing

Issue: https://github.com/joomla/joomla-cms/issues/20389

### Summary of Changes
Standard forms dont require inline js for validation, they only require a class `form-validate`!


### Testing Instructions

Apply patch and try to edit a module in the frontend. Save, Save and close and Close buttons should behave normally. Also check that editing the article also works

BTW, I've applied the same code to the rest views of com_config and also com_content

### Expected result

<img width="578" alt="screen shot 2018-05-14 at 20 37 53" src="https://user-images.githubusercontent.com/3889375/40016653-45bbfbdc-57b7-11e8-8030-b9db26c0a919.png">


### Actual result


### Documentation Changes Required
At some point we really need to document this submitbutton...


@wilsonge
